### ISSUE TITLE
Error message should specify the missing argument

### ISSUE THREAD

**Issue by mcf-rocks** [dusk-network/rusk#871]
value not present -- which value not present?

root@DUSK-DEV:~/rusk# ./target/release/rusk-node --kadcast_listen_address 127.0.0.1:9999 --kadcast_public_address 64.xxx.xxx.97:9999 --log-level info
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: NotPresent', node/src/chain/consensus.rs:52:61
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**To Reproduce**
compile the rusk binary and run it like above.

**Comment by Daksh14** [comment]
@mcf-rocks 

Thanks for reporting!

Will be closed by #879

**Comment by Daksh14** [comment]
On latest master

```
rusk on  master [✘!] via 𝗥 v1.71.0
➜ ./target/release/rusk-node --kadcast_listen_address 127.0.0.1:9999 --kadcast_public_address 64.xxx.xxx.97:9999 --log-level info
thread 'main' panicked at 'DUSK_CONSENSUS_KEYS_PASS not set: NotPresent', node/src/chain/consensus.rs:56:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

@ZER0 Good to close
